SU2_CFD/src/fluid/CSU2TCLib.cpp

```diff
@@ -857,10 +869,140 @@ vector<su2double>& CSU2TCLib::ComputeNetProductionRates(){
       if (iSpecies != nSpecies)
         ws[iSpecies] -= MolarMass[iSpecies] * (fwdRxn-bkwRxn);
     }
-  }
+
+    if (implicit) {
+      ChemistryJacobian(iReaction, V, eve, cvve, dTdU, dTvedU, val_jacobian);
```

Comment:
The values set in this function are used in ChemistryJacobian.  They aren't explicitly "set".  

This can be avoided if the Jacobian computations were within the ComputeNetProductionRates....I wasn't sure which method was more appropriate.